### PR TITLE
updatecli: init at 0.70.0

### PR DESCRIPTION
**Pull request by croissong**
## Description of changes

[Updatecli](https://updatecli.io/) is a command-line tool for declarative dependency management.
  
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by arthsmn — COMMENTED** [review]
Please use the [by-name category](https://github.com/NixOS/nixpkgs/tree/master/pkgs#category-hierarchy).

File: pkgs/development/tools/updatecli/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+buildGoModule rec {
+  pname = "updatecli";
+  version = "0.70.0";
+
+  src = fetchFromGitHub {
+    owner = "updatecli";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-MQoi/HvJqGCYzQLNsJul/7N3MXkV1X5d48InUSIWT8o=";
```
Comment (arthsmn) [inline pkgs/development/tools/updatecli/default.nix:17]:
```suggestion
    hash = "sha256-MQoi/HvJqGCYzQLNsJul/7N3MXkV1X5d48InUSIWT8o=";
```

**Review by arthsmn — APPROVED** [review]

**Comment by Croissong** [comment]
@arthsmn  Sorry if I'm missing something obvious - is there anything I can / need to do to get this merged?

**Review by h7x4 — APPROVED** [review]
Verified gpg key and github id, maintainer entry looks good.

The program seems to work correctly

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**